commit 38e923c104d37feec8e23b0c9348d6577a69188d
Author: jhenrique09 <jhenrique09.mcz@hotmail.com>
Date:   Fri Mar 27 16:55:56 2020 -0300

    vendor: Port face unlock feature [2/4]
    
    Also add a flag to opt-out
    
    @acroreiser: * simplified *

diff --git a/config/common.mk b/config/common.mk
index c3ff806..f77b46b 100644
--- a/config/common.mk
+++ b/config/common.mk
@@ -2,7 +2,6 @@
 $(call inherit-product-if-exists, vendor/extra/product.mk)
 $(call inherit-product-if-exists, vendor/lineage/config/crdroid.mk)
 $(call inherit-product-if-exists, vendor/addons/config.mk)
-$(call inherit-product-if-exists, external/moto/faceunlock/config.mk)
 
 PRODUCT_BRAND ?= crDroidAndroid
 
diff --git a/config/crdroid.mk b/config/crdroid.mk
index baa944d..1e9e7cb 100644
--- a/config/crdroid.mk
+++ b/config/crdroid.mk
@@ -35,6 +35,11 @@ PRODUCT_PACKAGES += \
     OmniStyle \
     Longshot
 
+# Face Unlock
+TARGET_FACE_UNLOCK_SUPPORTED := true
+PRODUCT_SYSTEM_DEFAULT_PROPERTIES += \
+    ro.face.moto_unlock_service=$(TARGET_FACE_UNLOCK_SUPPORTED)
+
 # Fonts
 PRODUCT_PACKAGES += \
     fonts_customization.xml \
